communication/slack-config/usergroups.yaml

```diff
@@ -56,8 +56,9 @@ usergroups:
       - joelsmith
       - lukehinds
       - micahhausler
+      - ritazh
+      - SaranBalaji90
```

Comment:
entries for them have to be added here:
https://github.com/kubernetes/community/blob/master/communication/slack-config/users.yaml

Since slack allows overlapping names, we map the friendly name to their actual slack ID

Reply:
@mrbobbytables I think this should be good now :)